data/vm-sizes-list.json

```diff
@@ -161,6 +161,9 @@
         "standard_nv6_promo",
         "standard_nv12_promo",
         "standard_nv24_promo",
+        "standard_nv12s_v3", 
+        "standard_nv24s_v3", 
+        "standard_nv48s_v3",
```

Comment:
both PAAS and IAAS I beleive

Reply:
for both families

Reply:
Jake told me the HBv2 should be IAAS only but I included those three for PAAS as well